index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta name="" content="">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+  </head>
+  <body>
+    <nav class="top_bar">
+      <img id="nav" src="/Images/App_Bar.png" alt="" />
+    </nav>
+    <div id="background">
+      
+    <div id="intro">
+    <h1 id="greeting">Hey there, I am Solly</h1>
+    <h3 id="profession">I am a Software Developer</h3>
+    <p class="description">I can build a product, feature or website for you. Kindly look at some of my work and experience!. If what you see is desirable and need project coded, don't hesitate to contact me</p>
+    </div>
+    <div id="image_div">
+      <img class="images" src="/Images/Github_icon.png" alt="A Github icon" />
+      <img class="images" src="/Images/LinkedIn_icon.png" alt="A LinkedIn_icon" />
+      <img class="images" src="/Images/Twitter_icon.png" alt="A Twitter icon" />
+    </div>
```

Comment:
- [ ] [REQUIRED] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon should has a link (`a` tag). Please make changes!